src/permutation.rs

```diff
@@ -116,9 +116,14 @@ macro_rules! permutation_trait_impl(
           // Have all parsers (including void) failed?
           if l == input.len() {
             // Skip unknown Element if possible.
-            if let Error { id: _, kind: crate::ebml::ErrorKind::MissingElement } = err {
-              if let Ok((i, id)) = crate::ebml::skip_master(input) {
-                log::warn!("Skipped unknown Element 0x{id:X}");
+            if let Error { id: _, kind: $crate::ebml::ErrorKind::MissingElement } = err {
+              if let Ok((i, id)) = $crate::ebml::skip_element(input) {
```

Comment:
Is it possible to use `if let` chains here?

They should be stabilized starting from Rust 1.64 https://github.com/rust-lang/rust/pull/94927

Reply:
Oops, doesn't work yet. Some stuff is still not stable:
https://github.com/rust-lang/rust/issues/53667

```
 1  error[E0658]: `let` expressions in this position are unstable                                                          ▐
    --> src/permutation.rs:119:16                                                                                          ▐
     |                                                                                                                     ▐
 119 |               if let Error { id: _, kind: $crate::ebml::ErrorKind::MissingElement } = err                           ▐
     |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                           
```

Reply:
Oh, what a pity! Ok, no problem then